nncf/tensorflow/graph/transformations/layout.py

```diff
@@ -116,15 +118,18 @@ def _find_transformation(self,
 def check_target_points(tp0: TargetPoint, tp1: TargetPoint) -> bool:
     return tp0.type in GRAPH_NODE_TYPES and \
            tp1.type in GRAPH_NODE_TYPES and \
+           isinstance(tp0, TFLayerPoint) and isinstance(tp1, TFLayerPoint) and \
            tp0.layer_name == tp1.layer_name
 
 
 def is_object_removed(removed_target: TargetPoint, command_target: TargetPoint) -> bool:
-    layer_removed = removed_target.type == TargetType.LAYER and \
+    layer_removed = isinstance(removed_target, TFLayerPoint) \
```

Comment:
What reason of this changes?

Reply:
The `.type` comparison is just a glorified `isinstance` call anyway, but `isinstance` allows to use IDE support for attribute completion in the code that follows `isinstance.

Reply:
Restored the check for the `removed_target` type. The `command_target` type is checked even in this version a couple of lines below.